CONTRIBUTING.md

```diff
@@ -2,6 +2,52 @@
 
 Contact us on our Discord at [https://discord.gg/YjK2JAD](https://discord.gg/YjK2JAD)
 
-Feel free to make pull requests or issues, but please note that we are currently focused on making the mod, not adding more content.
+Feel free to make pull requests or issues, but please note that we are currently focused on making the mod functional, before adding more content.
 
-People who make pull requests should be familiar with the mod licenses; if you make a PR, you agree to the license.
+## PR Guidelines
+
+* Keep PR's to one topic, if possible. This permits us to use version control to find software regressions more easily.
```

Comment:
It's far more important to keep *commits* to one topic.

Reply:
True.